libraries/standard/mqtt/include/mqtt.h

```diff
@@ -55,13 +58,11 @@ typedef uint32_t (* MQTTGetCurrentTimeFunc_t )( void );
  *
  * @param[in] pContext Initialized MQTT context.
  * @param[in] pPacketInfo Information on the type of incoming MQTT packet.
- * @param[in] packetIdentifier Packet identifier of incoming PUBLISH packet.
- * @param[in] pPublishInfo Incoming PUBLISH packet parameters.
+ * @param[in] pDeserialized Deserialized information from incoming packet.
```

Comment:
Now that this PR updates this event callback, can you add documentation to mention that the event callback would only be called for either `MQTTSuccess` or `MQTTServerRefused` de-serialization statuses? (May be good to also mention that `MQTTServerRefused` is only passed for `SUBACK` packet events)
I ask so that we don't forget to mention that during whenever the documentation task for all MQTT library is taken up...

Reply:
Added a `@note`